src/ImageSharp/Memory/BufferArea{T}.cs

```diff
@@ -45,11 +45,26 @@ public BufferArea(IBuffer2D<T> destinationBuffer)
         /// </summary>
         public Size Size => this.Rectangle.Size;
 
+        /// <summary>
+        /// Gets the width
+        /// </summary>
+        public int Width => this.Rectangle.Width;
+
+        /// <summary>
+        /// Gets the height
+        /// </summary>
+        public int Height => this.Rectangle.Height;
+
         /// <summary>
         /// Gets the pixel stride which is equal to the width of <see cref="DestinationBuffer"/>.
         /// </summary>
         public int Stride => this.DestinationBuffer.Width;
 
+        /// <summary>
+        /// Gets a value indicating whether the area refers to the entire <see cref="DestinationBuffer"/>
+        /// </summary>
+        public bool IsFullBufferArea => this.Size == this.DestinationBuffer.Size();
+
         /// <summary>
         /// Gets or sets a value at the given index.
         /// </summary>
```

Comment:
*Diff wouldn't let me highlight where I wanted to*

`GetReferenceToOrigo` I'm assuming this is a typo? `GetReferenceToOrigin` perhaps?

Reply:
It's just my Hunglish :D

Reply:
😄 Haha... Far better than any language skills I've accumulated.